tools/reviewer/web/src/app/dashboard/reviews-panel/diff/diff.component.ts

```diff
@@ -1,14 +1,26 @@
+// NOTICE: it's actually not a diff, it's a two files changes
+// TODO: rename the component and linked files
```

Comment:
Rename to what?

Reply:
e.g. `FileChangeComponent`

Reply:
Good. Can you add that as a suggestion to the TODO?

Reply:
@oferb already did